### PR TITLE
Rb 4.1.x fbcp v2

### PR DESCRIPTION
**Pull request by ian57**
Please make sure your PR is ready to be merged !

- [x] You added the changes in CHANGELOG.md
- [x] You choose the right repository branch to make the PR
- [x] You described the PR as below

Changes :
- added the fbcp program to allow display on small tftscreen
- added tft waveshare 3.2 and 3.5 overlays for tft screens
- added  S13fbcp and its management from recalbox.conf

### CODE REVIEW

File: package/rpi-fbcp/S11fbcp [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
```
Comment (substring) [inline package/rpi-fbcp/S11fbcp:2]: You should add a `system.fbcp.enabled` (or such a name) in recalbox.conf and do like https://github.com/recalbox/recalbox-buildroot/blob/master/board/recalbox/fsoverlay/etc/init.d/S95api#L13 + https://github.com/recalbox/recalbox-buildroot/blob/master/board/recalbox/fsoverlay/etc/init.d/S95api#L45-L46

**Review by substring — CHANGES_REQUESTED** [review]

File: package/rpi-fbcp/S11fbcp (hunk 1 above)
Comment (ian57) [reply package/rpi-fbcp/S11fbcp:2]: Yep, I know... I think I will bother you with this stuff to get informations about how the configuration works from recalbox.conf to config file such config.txt :)

**Comment by ian57** [comment]
Ok fbcp management in recalbox.conf is done.

**Review by substring — APPROVED** [review]

**Comment by digitalLumberjack** [comment]
Thank you @ian57, @substring

**Pull request merged**